ssm_jax/hmm/models/multinomial_hmm.py

```diff
@@ -35,16 +52,17 @@ def __init__(self,
         num_classes = emission_probs.shape[2]
         self._num_trials = num_trials
         self._emission_probs = Parameter(emission_probs, bijector=tfb.Invert(tfb.SoftmaxCentered()))
-        self._emission_prior_concentration = Parameter(emission_prior_concentration  * jnp.ones(num_classes),
+        self._emission_prior_concentration = Parameter(emission_prior_concentration * jnp.ones(num_classes),
                                                        is_frozen=True,
                                                        bijector=tfb.Invert(tfb.Softplus()))
 
     @classmethod
     def random_initialization(cls, key, num_states, num_emissions, num_classes, num_trials=1):
-        key1, key2, key3 = jr.split(key, 3)
+        key1, key2, key3, key4 = jr.split(key, 4)
         initial_probs = jr.dirichlet(key1, jnp.ones(num_states))
         transition_matrix = jr.dirichlet(key2, jnp.ones(num_states), (num_states,))
         emission_probs = jr.dirichlet(key3, jnp.ones(num_classes), (num_states, num_emissions))
+        num_trials = jr.randint(key4, shape=(1,), minval=0, maxval=10).astype(jnp.float32)[0]
```

Comment:
`num_trials` is given as a keyword argument. I think it makes sense for the user to specify this since it's a constant, like `num_states`, etc.

Reply:
+1

Reply:
Done.